_data/simple-icons.json

```diff
@@ -13891,6 +13891,11 @@
             "hex": "E60000",
             "source": "https://web.vodafone.com.eg"
         },
+        {
+            "title": "Void Linux",
+            "hex": "478061",
+            "source": "https://commons.wikimedia.org/wiki/File:Void_Linux_logo.svg"
```

Comment:
```suggestion
            "source": "http://alpha.de.repo.voidlinux.org/logos/void-dark2.svg"
```

Reply:
I was on vacation, thanks for the review and the change makes sense to me!